cookiecutter/{{cookiecutter.github_repo_name}}/CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to {{cookiecutter.title}}
+
+## Build
```

Comment:
I'd suggest adding a section labelled `Python` and adding the python related documentation under it.

Reply:
I think for now it will make the template and instructions more complicated, what about adding it when we add Rust support.